setup.py

```diff
@@ -5,8 +5,7 @@
 import stat
 import sys
 
-#VERSION="2.1dev4"
```

Comment:
This is an important line to be able to remember how the format for development versions is.
Please leave it in there ;-)

Reply:
Maybe we should drop hard-coding of version numbers and use git tags entirely for versioning.

Reply:
Fixed in 67e3d62